modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-sets/FieldSets.es.js

```diff
@@ -28,6 +28,36 @@ import FieldSetModal from './FieldSetModal.es';
 import useDeleteFieldSet from './actions/useDeleteFieldSet.es';
 import usePropagateFieldSet from './actions/usePropagateFieldSet.es';
 
+function getSortedFieldsets(fieldsets) {
+	return fieldsets.sort((a, b) => {
+		const localizedValueA = getLocalizedValue(a.defaultLanguageId, a.name);
+		const localizedValueB = getLocalizedValue(b.defaultLanguageId, b.name);
+
+		return localizedValueA.localeCompare(localizedValueB);
+	});
+}
+
+function getFielteredFieldsets(fieldsets, keywords) {
```

Comment:
This function name, could be getFilteredFieldsets ?

Reply:
Yes, I'll update it